app/controllers/claims/schools/mentors_controller.rb

```diff
@@ -42,10 +45,18 @@ def set_mentor
     @mentor = @school.mentors.find(params.require(:id))
   end
 
+  def set_mentor_membership
+    @mentor_membership = @mentor.mentor_memberships.find_by!(school: @school)
+  end
```

Comment:
This logic is duplicated on line 36. We should remove line 36 and use this reference instead.